apps/products/models.py

```diff
@@ -0,0 +1,43 @@
+from django.core.validators import MaxValueValidator, MinValueValidator
+from django.db import models
+
+from apps.users.models import User
+
+
+# Create your models here.
+
+
+class BaseModelMixin(models.Model):
```

Comment:
Nu creat cate un base model in fiecare applicatie. Cream unul singur in apllicatia "common"